CSharp/grades/grades.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">

```

Comment:
dll files should not be included in commits

Reply:
already solved